web-common/src/layout/overlay-store.ts

```diff
@@ -7,4 +7,4 @@ interface Overlay {
   message?: string;
 }
 
-export const overlay = writable<Overlay>(null);
+export const overlay = writable<Overlay | null>(null);
```

Comment:
This is explicitly set to null in web-common/src/features/sources/workspace/SourceWorkspaceHeader.svelte, so this type is needed